src/main/cpp/t07_ip.cpp

```diff
@@ -34,5 +34,18 @@
 using namespace std;
 
 int t07_ip() {
-
+string b;
+int a, k = 0, prevk=0;
+getline(cin, b);
+a =  b.size();
+for (int i = 0; i < a; i++)
+{if (b[i] == '.'){
+if(((b.substr(prevk, i - prevk) > "255" and b.substr(prevk, i - prevk).size() == 3) or b.substr(prevk, i - prevk) < "0") or b.substr(prevk, i - prevk).size() > 3)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comparison of a boolean expression with an integer other than 0 or 1.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4494122)